src/main/java/com/powsybl/openloadflow/network/LfAction.java

```diff
@@ -335,7 +335,9 @@ public void apply(LfNetworkParameters networkParameters) {
         if (generatorChange != null) {
             LfGenerator generator = generatorChange.generator();
             if (!generator.isDisabled()) {
-                generator.setTargetP(generator.getTargetP() + generatorChange.deltaTargetP());
+                double deltaTargetP = generatorChange.deltaTargetP();
```

Comment:
First, thank you for the bug fix. Second, your fix proposal is not very adapted for long-term support and users. I think that you have to do if loops in order to add a log message to explain why we don't want to modify a target P outside of the generator active limits. Because the fact is that you are not going to apply the remedial action asked by the user... we can change the remedial action and log, or throw...

Reply:
I agree to add a log to explain that the whole action has not been applied (and why). Throwing an error seems problematic to me. In case of relative redispatch action, with big slack distribution in pre-contingency case, this case could happen a lot.

Reply:
need indeed a log

should also probably make use of option `useActiveLimit` of LfNetworkParameters, as follows:
- `useActiveLimit` set to `false`: ignore limits
- `useActiveLimit` set to `true`:
  - generator is within [Pmin;Pmax] before the action: restrict the change and log warning
  - generator is already outside [Pmin;Pmax]: ignore limits